src/index.html

```diff
@@ -0,0 +1,34 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+    <title>SAM To Do List</title>
+</head>
+<body>
+<section class="container">
+
+    <div class="header">
+        <h1 class="title">Today's To Do</h1>
+        <i class="bi bi-arrow-repeat"></i>
+    </div>
+
+    <div class="add-item">
+        <input type="text" class="list-input" placeholder="Add to your list...">
+        <button class="add-button">
+        <i class="bi bi-plus"></i>
+        </button>
+    </div>
+
+    <ul class="add-list" id="add-list">
+    
+    </ul>
+
+    <button type="button" class="clearBtn">
+        <span>Clear all completed</span>
+    </button>
+    
+</section>
```

Comment:
- [ ] @samuelkinyanjui Great work on your todo list, it would be nice if you could consider increasing the width to match the design given, 
- [ ] Also the margin between the  checkbox and todo description and the distance between the tasks and the `Clear all completed` button is too big kindly consider reducing it to  the match the design given

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/samuelkinyanjui/ToDolistsm/assets/61408860/04aa0519-a372-41e7-bace-30f40bfd3041)  |  ![image](https://github.com/samuelkinyanjui/ToDolistsm/assets/61408860/c499ef0b-8e84-48c8-9212-987e311aac20)

Reply:
You did well in increasing the size of your todo, kindly reduce the margins as instructed by the previous reviewer as you are advised to have a design that is closely related to your given framework

Your design :
![Uploading image.png…]()

Reply:
Kindly  i cannot see the image you intended to attach